torch/_inductor/codegen/triton.py

```diff
@@ -2918,6 +2918,7 @@ def inductor_meta_common():
             "min_split_scan_rblock": config.triton.min_split_scan_rblock,
             "spill_threshold": config.triton.spill_threshold,
             "store_cubin": config.triton.store_cubin,
+            "compile_id": str(torch._guards.CompileContext.current_compile_id()),
```

Comment:
This is necessary if we want the compile_id to show up in the tlparse / perfetto metadata. I assume we do.